extensions/banner/runtime/src/main/java/io/quarkus/banner/runtime/BannerRecorder.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.banner.runtime;
+
+import io.quarkus.runtime.annotations.Recorder;
+import org.jboss.logging.Logger;
+
+
+@Recorder
+public class BannerRecorder {
+    public void print(String banner) {
+        Logger LOGGER = Logger.getLogger(BannerRecorder.class.getPackage().getName());
+        LOGGER.debug(banner);
```

Comment:
`info` level is more adequate here, also make the `LOGGER` variable all lowercase, to conform to the Java coding standard

Reply:
I checked Loggers in other files those were capital. Sure I will change it

Reply:
Probably because they were declared as static final attributes, and even in this case they should be lowercased: https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case